packages/Footer/Footer.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+  <section id="footer">
+    <slot>
```

Comment:
this slot is not really useful as it overrides the whole component, let's remove it

Reply:
Fixed.